specs/IsNonClientRegionSupportEnabled.md

```diff
@@ -0,0 +1,176 @@
+{
+    var coreWebView2Settings = webView.CoreWebView2.Settings;
+    var urlCompareExample = "www.microsoft.com";
+    var uri = new Uri(args.Uri);
+
+    if (String.Equals(uri.Host, urlCompareExample, StringComparison.OrdinalIgnoreCase) &&
+        !coreWebView2Settings.IsNonClientRegionSupportEnabled)
+    {
+        coreWebView2Settings.IsNonClientRegionSupportEnabled = true;
+    }
+    else if (!String.Equals(uri.Host, urlCompareExample, StringComparison.OrdinalIgnoreCase) && 
+        coreWebView2Settings.IsNonClientRegionSupportEnabled)
+    {
+        coreWebView2Settings.IsNonClientRegionSupportEnabled = false;
+    }
+}
+```
+
+## Declaring Non-client App Regions
+Non-client regions are HTML elements that are marked with the css style `app-region`.
+* Draggable regions can be declared through the values `drag` or `no-drag`. 
+    * `app-region: drag` will support [draggable region functionality](#description) for the html element.
+    * `app-region: no-drag` will change cursor to I-bar, with text highlighting enabled. 
+    Elements with this style will not support draggable region functionality.
+```html
+<!DOCTYPE html>
+<body>
+    <div style="app-region:drag">Drag Region</div>
+    <div style="app-region:no-drag">No-drag Region</div>
+</body>
+<html>
+```
+
+# Remarks
+If the feature flag (`msWebView2EnableDraggableRegions`) is used to enable the CSS style `app-region`'s
+values `drag` and `no-drag`, draggable region support will remain enabled 
+even if the `IsNonClientRegionSupportEnabled` setting is `FALSE`. 
+* Note: The feature flag is experimental and should not be used in production.
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+## Win32 C++
+```cpp
+
+[uuid(436CA5E2-2D50-43C7-9735-E760F299439E), object, pointer_default(unique)]
+interface ICoreWebView2Settings12 : ICoreWebView2Settings11 {
+  /// The `IsNonClientRegionSupportEnabled` property enables web pages to use the 
+  /// `app-region` CSS style. Changing the `IsNonClientRegionSupportEnabled` property
+  /// takes effect after the next navigation of the top level document. Defaults to `FALSE`.
+  /// 
+  /// When this property is `TRUE`, then all the following non-client region support 
+  /// will be enabled:
+  /// 1. Draggable Regions will be enabled and treated as a window's title bar. 
+  /// Draggable Regions are non-client regions on a webpage that are exposed through the css
+  /// attribute `app-region` and can take the values `drag` or `no-drag`. When set to 
+  /// `drag`, these regions will be treated like the window's title bar, supporting 
+  /// dragging of the entire WebView and its host app window, the title bar context menu
+  /// upon right click, and the maximizing to fill the window and restoring the window size
+  /// upon double click of the html element. 
+  ///
+  /// When set to `FALSE`, then all non-client region support will be disabled. Web
```

Comment:
Not exactly true. If the feature flag is enabled, then setting this to FALSE has no effect, and non-client region support is still enabled.

Reply:
Please fix.

Reply:
fixed